doc/changes/names.inc

```diff
@@ -298,4 +298,6 @@
 
 .. _Adam Li: http://github.com/adam2392
 
-.. _Ramiro Gatti: https://github.com/ragatti
\ No newline at end of file
+.. _Ramiro Gatti: https://github.com/ragatti
+
+.. _Svea Marie Meyer: https://github.com/SveaMeyer13
```

Comment:
Name here and above need to match, it's what CircleCI complains about (search "warning" in the log and you'll see it)

https://app.circleci.com/pipelines/github/mne-tools/mne-python/897/workflows/179f9067-911b-4472-b7eb-20702ffa91f8/jobs/19252

And the broken link is visible here

https://19252-1301584-gh.circle-artifacts.com/0/dev/whats_new.html